### DISCUSSION TITLE
My ender 3 v2 bed size incorrect

### DISCUSSION THREAD

**Discussion by Chrisyahya** [Jyers/Marlin#407] [Q&A] [answered]
Hello I wanna ask.I just installed 1.3.0 version,all superb,but one question

Why is the dimension size says 230 230 250?

Ain't that ender 3 v2 is 220 220 250?

![IMG_20210408_113435](https://user-images.githubusercontent.com/60070242/113971199-86294280-9862-11eb-8b44-e897a9512ac0.jpg)

**Comment by smooreace** [comment]
Isn't?

No, the real dimensions are 230 x 230

**Reply by Chrisyahya** [reply]
..so you mean i could print a 230x230 size?

**Comment by GuidoHendriks** [comment, marked as answer]
Manufacturer says it's 220x220, but the actual bed size is 235x235.

**Reply by Chrisyahya** [reply]
..so you mean i could print a 230x230 size?

**Reply by GuidoHendriks** [reply]
In width I think it's possible to go full 230, in height please make sure you stay clear of the bed clips.

Try manually moving it in the menu to 230 on both ends and see where the nozzle is. Might run of the bed depending on home offsets.

**Reply by Raiden911** [reply]
I actually think this is why my printer is having issues with BI touch. 

When it auto levels it'll hit the center first, then home in on the x and y axis. Then the BI touch will fall outside of the build plate. It attempts to home in on Z, but because the BI is outside of the build plate it just bottoms out (it does show homing 1 out of 25  completed). Then it gives up and says it's completed.

I think because of this it wont home properly. Latest firmware from Creality has no issues (Ender-3 V2-Marlin-2.0.1-HW-4.2.2-mainboard-V1.1.2-Compatible with BLTouch and filament detection).. But I'm posting here cause I like this configuration better

**Comment by smooreace** [comment]
You can easily measure your bed. You probably don't have the stock Y
carriage installed?
Did you properly add your probe X and Y offsets? You can also compile your
own firmware with the smaller 220x220 bed size, as well as other options.

But again, if you measure your bed you see it's clearly 234.9mm X 234.9mm.
So 230x230 gives you room on all sides

**Comment by aryajuanda** [comment]
i love latest version with 230x230 bed size  + 15 on X Max it give 245 om x move so my bltouch can reach around 200 mm from left bed (my probe to nozzle) offset is -45, -8, 0) so i can get more accurate read on Auto Bed Leveling because it can reach far right...
with 20 Margin it will give ABL from 20 - 200 on X.. 
X on 230 mm nozzle still in the bed (+- 5 mm from the end)
